Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04 AS builder
+FROM ubuntu:22.04 AS builder
```

Comment:
Maybe `python:3.10-slim-bookworm` would be better. `python3-dev` and `python3-pip` packages are already there.